samples/snippets/csharp/api/system.reflection.metadata.ecma335/metadatabuilder/MetadataBuilderSnippets.cs

```diff
@@ -0,0 +1,224 @@
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Reflection.Metadata.Ecma335;
+using System.Reflection;
+
+namespace MetadataBuilderSnippets
+{
+    static class MetadataBuilderSnippets
+    {
+        //<SnippetEmitConsoleApp>
+        private static readonly Guid s_guid = new Guid("87D4DBE1-1143-4FAD-AAB3-1001F92068E6");
+        private static readonly BlobContentId s_contentId = new BlobContentId(s_guid, 0x04030201);
+        
+        private static MethodDefinitionHandle EmitHelloWorld(MetadataBuilder metadata, BlobBuilder ilBuilder)
+        {
+            // Create module and assembly for a console application.
+
+            metadata.AddModule(
+                0,
+                metadata.GetOrAddString("ConsoleApplication.exe"),
+                metadata.GetOrAddGuid(s_guid),
+                default(GuidHandle),
+                default(GuidHandle));
+
+            metadata.AddAssembly(
+                metadata.GetOrAddString("ConsoleApplication"),
+                version: new Version(1, 0, 0, 0),
+                culture: default(StringHandle),
+                publicKey: default(BlobHandle),
+                flags: 0,
+                hashAlgorithm: AssemblyHashAlgorithm.None);
+
+            // Create references to System.Object and System.Console types.
+
```

Comment:
NIT: Please remove the lines here and everywhere else below, not gonna comment on others
```suggestion
```